Assets/HoloToolkit/Input/Scripts/Focus/InputSourcePointer.cs

```diff
@@ -19,6 +19,8 @@ public class InputSourcePointer : IPointingSource
 
         public BaseRayStabilizer RayStabilizer { get; set; }
 
+        public Cursor CursorOverride { get; set; }
```

Comment:
I don't think this should be an override, but instead have the InputSourcePointer own the Cursor

Reply:
With MultiPointerSelector.cs (as opposed to SimpleSinglePointerSelector.cs) it's a little more intuitive to opt-in to unique cursors. But either way works.